### PR TITLE
feat: wire up multi hop ui elements

### PR DESCRIPTION
**Pull request by woodenfurniture**
## Description

Wires up ui elements of multi hop stepper to actual quote data.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

NA

## Risk

Includes very minor change to `useTradeExecution` hook which may present low risk of broken trade execution with multi hop toggled off.

## Testing

Check the existing trade exectution (no multi hop)

### Engineering

☝️

### Operations

☝️

## Screenshots (if applicable)

image

### CODE REVIEW

**Comment by woodenfurniture** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5586** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5586" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5586?utm_source=stack-comment).

File: src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/steps.tsx [hunk 1]
```diff
@@ -22,24 +25,88 @@ import { RawText, Text } from 'components/Text'
...
+    case KnownChainIds.BitcoinCashMainnet:
+      return 'BCH'
+    case KnownChainIds.CosmosMainnet:
+      return 'COSM'
+    case KnownChainIds.ThorchainMainnet:
+      return 'THOR'
+    case KnownChainIds.DogecoinMainnet:
+      return 'DOGE'
+    case KnownChainIds.LitecoinMainnet:
+      return 'LTC'
```
Comment (woodenfurniture) [inline src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/steps.tsx:92]: appreciate any input or suggestions to correct or improve these

File: src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/steps.tsx [hunk 2]
```diff
@@ -170,12 +242,14 @@ export const getHopSummaryStep = ({
...
   const tradeType = buyAssetChainId === sellAssetChainId ? TradeType.Swap : TradeType.Bridge
+  const sellChainSymbol = getChainShortName(sellAssetChainId as KnownChainIds)
+  const buyChainSymbol = getChainShortName(buyAssetChainId as KnownChainIds)
   return {
     title:
       tradeType === TradeType.Swap
         ? `${tradeType} on ${sellChainName} via ${swapperName}`
         : `${tradeType} from ${sellChainName} to ${buyChainName} via ${swapperName}`,
-    description: `${sellAmountCryptoFormatted}.ETH -> ${buyAmountCryptoFormatted}.AVA`, // TODO: chain "symbol"
+    description: `${sellAmountCryptoFormatted}.${sellChainSymbol} -> ${buyAmountCryptoFormatted}.${buyChainSymbol}`, // TODO: chain "symbol"
```
Comment (0xApotheosis) [inline src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/steps.tsx:252]: Is this TODO still relevant?

File: src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/steps.tsx (hunk 1 above)
Comment (0xApotheosis) [reply src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/steps.tsx:92]: These all seem reasonable for now!

File: src/components/MultiHopTrade/hooks/useTradeExecution/useTradeExecution.ts [hunk 3]
```diff
@@ -49,6 +45,8 @@ export const useTradeExecution = ({
     selectPortfolioAccountMetadataByAccountId(state, { accountId: sellAssetAccountId }),
   )
 
+  const swapperName = useAppSelector(selectActiveSwapperName)
+  const tradeQuote = useAppSelector(selectActiveQuote)
```
Comment (0xApotheosis) [inline src/components/MultiHopTrade/hooks/useTradeExecution/useTradeExecution.ts:49]: 🧠

**Review by 0xApotheosis — APPROVED** [review]
gm

File: src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/steps.tsx (hunk 2 above)
Comment (woodenfurniture) [reply src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/steps.tsx:252]: well spotted, fixing

**Pull request merged**